### ISSUE TITLE
[Feature] Variable lables viewer [sf#185]

### ISSUE THREAD

**Issue by openMSX-import** [openMSX/debugger#33]
**Reported by sd-snatcher on 2009-04-20 21:41 UTC**
The "Variable lables" viewer would be a dedicated viewer for labels. You could add/remove many labels \(and the data size, off course\) and watch their contents. That would be very usefull for debugging, as normally your data can be scattered all over the RAM, which requires you to have a lot of "Debuggable Hex Views" opened.

**Comment by MBilderbeek** [comment]
Sounds like what we have as the RAM watch script in openMSX itself. Very useful for a debugger GUI as well indeed.

**Comment by bengalack** [comment]
Yes, would be very useful to have a small window with a set of "variables" listed. Even values from dereferenced pointer-to-pointer-to-char would have been great :) This is not possible in ram_watch today.

**Comment by MBilderbeek** [comment]
Isn't this available in the openMSX built-in debugger now?

**Comment by m9710797** [comment]
The "Watch expression" window in openMSX can indeed do all this (and more).
I'll close this ticket.
